packages/bierzo-wallet/src/routes/receivePayments/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChainId, Identity } from '@iov/bcp';
+import { TransactionEncoder } from '@iov/encoding';
+import React from 'react';
+import * as ReactRedux from 'react-redux';
+
+import { history } from '..';
+import { getChainName } from '../../config';
+import { getCodecForChainId } from '../../logic/codec';
+import { RootState } from '../../store/reducers';
+import { BALANCE_ROUTE } from '../paths';
+import Layout, { ChainAddressMap } from './components';
+
+function onReturnToBalance(): void {
+  history.push(BALANCE_ROUTE);
+}
+
+const ReceivePayment = (): JSX.Element => {
+  const pubKeys = ReactRedux.useSelector((state: RootState) => state.extension.keys);
+
+  const [chainAddressMap, setChainAddressMap] = React.useState<ChainAddressMap[]>([]);
+
+  React.useEffect(() => {
+    async function processAddresses(pubKeys: { [chain: string]: string }): Promise<void> {
+      if (!pubKeys) return;
```

Comment:
This check sould not be necessary. `pubKeys` type does not allow a falsy value

Reply:
Yes, but check fails because of this.

Reply:
Anyway this was fixed